src/twisted/newsfragments/7530.bugfix

```diff
@@ -0,0 +1 @@
+Allow for key and cert .pem files that do not have a a trailing newline
```

Comment:
I guess I just misunderstood the change :).  But my original point still stands: this change is to a specific area of Twisted (the `ssl:` endpoint, I believe) but the changelog entry here is phrased as if it's universal (i.e. as if nothing that previously parsed a "key" or "cert" PEM file would work if it didn't have a trailing newline).

Since you've correctly explained the change here (the bug doesn't exist elsewhere), please just update the news fragment to be clearer.  The standard for writing them does say to use the FQPN to identify the affected code, although here the endpoint prefix is probably what users really want to know about.